react/features/subtitles/middleware.ts

```diff
@@ -231,7 +247,31 @@ function _endpointMessageReceived(store: IStore, next: Function, action: AnyActi
             // after the stable part.
             newTranscriptMessage.unstable = text;
         }
+    }
+
+    if (newTranscriptMessage) {
+        if (newTranscriptMessage.final) {
```

Comment:
You can combine both conditions into one to avoid the extra indentation and make it more legible.

Reply:
Generally yes! But the problem is that we want to update the redux with the new action for all types of messages and not only for the final ones. I can combine those and then surround the update of the messages in a new if but that would look uglier IMHO

Reply:
ah I had missed the end, good point!